example-config.yml

```diff
@@ -15,4 +15,4 @@ apps:
     hostname: www.local.bcc3d.ca
     google_analytics_key: UA-162929587-1
     google_maps_api_key:
-    jekyll_environment: #production
\ No newline at end of file
+    production: true
```

Comment:
default was "false" before, which I think is correct, could you change it here?
ready for merge and deploy after this

Reply:
Definitely! Sorry about that, will do the change now.